source/systems/cloud_user_guide.rst

```diff
@@ -117,6 +118,292 @@ This includes data stored in a CSP public, free to access repositories, like
+
+- Lustre: object storage for backup and restore and output files
+- Bucket/blob storage: a container for objects.
+- Contrib file system: a project’s custom software library.
+
+.. note::
+
+  An "object" is a file and any metadata that describes that file.
+
+Lustre file system
+^^^^^^^^^^^^^^^^^^
+
+Lustre is a parallel file system, available as ephemeral and persistent storage
+on the AWS, Azure, and GCP cloud platforms. A lustre file system can be
+attached and mounted on a cluster, and is accessible only from an active
+cluster. To create a lustre file system, access the Storage tab, and click Add
+Storage. You can create any number of lustre file systems. See `this article
+<https://parallelworks.com/docs/storage/creating-storage>`_ for
+information on creating a storage link.
+
+Bucket/Block blob storage
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Bucket storage and Block blob storage are containers for objects.
+An object is a file and any metadata that describes that file.
+Metadata can include use cases, such as
+data lakes, websites, mobile applications, backup and restore, archive,
+enterprise applications, IoT devices, or big data analytics. On AWS and GCP,
+the storage is called S3 bucket, and bucket respectively, whereas in Azure, the
+storage used is Block blob storage, which functions as a bucket and an NFS
+storage. Pricing information is available at this `link <https://aws.amazon.com/s3/pricing/>`_ .
+Projects using AWS, and GCP platforms can create as many buckets as needed, and
+mount them on a cluster. The project’s default bucket is accessible from the
+public domain using the keys.
+
+Contrib file system
+^^^^^^^^^^^^^^^^^^^
+
+The Contrib file system concept is similar to on-premise contrib. It is used to
+store files for team collaboration. You can use this storage to install custom
+libraries or user scripts.
+
+AWS Contrib storage charge is $0.30 per GB per Month. The cost is calculated
+based on the storage usage. Both AWS and Azure charge based on the usage, with
+a pay-as-you-go model like your electricity bill.
+
+GCP charges on allocated storage, so whether the storage is used or not, the
+project pays for the provisioned capacity. The default provisioned capacity of
+Google Cloud contrib file system is 2.5 TiB, costs $768.00 per month. The
+contrib volume can be removed from a project by request. Send email to
+rdhpcs.cloud.help@noaa.gov, with Remove Contrib Volume in the subject.
+
+
+Cloud Project Management: Create a Cloud Project
+------------------------------------------------
+
+.. note::
+
+  Cloud projects are specific to a Cloud platform. The platform is indicated by the
+  prefix in the project name (ca- for AWS, cz- for Azure, cg- for GCP).
+
+Cloud projects are defined thorugh the AIM system. Before you can create a
+project in AIM, it must have assigned allocation. Allocations are approved by
+the NOAA RDHPCS allocation committee.
+
+To verify the allocation amount for your project, create a cloud help desk
```

Comment:
Rewrite the paragraph:

To verify the allocation amount for your project, create a cloud help desk
ticket to schedule a meeting. Send email to rdhpcs.cloud.help@noaa.gov, with
Allocation for <Project> in the subject line. A SME can help you translate your
business case into an allocation estimate.

as 

If your project is large in size and requires assistance in capacity planning, planning and porting, open a help desk ticket by sending an email to rdhpcs.cloud.help@noaa.gov.